templates/run_log_rotation.sh.j2

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+LOG_PATH="{{ log_path }}"
```

Comment:
Why to define this env. variable? Does the `logrotate` process needs it?

Reply:
good catch
a6f3ad45